File: span-builder/src/main/java/com/maciega/bartosz/span_builder/spans/builders/AbsoluteSizeSpanBuilder.java
package com.maciega.bartosz.span_builder.spans.builders;

import android.os.Parcel;
import android.text.style.AbsoluteSizeSpan;

/**
 * Created by bartoszmaciega on 03/03/17.
 */

public class AbsoluteSizeSpanBuilder extends AbstractSpanTypeBuilder<AbsoluteSizeSpan> {

    public AbsoluteSizeSpanBuilder(int size, boolean dip) {
        span = new AbsoluteSizeSpan(size, dip);
    }

    public AbsoluteSizeSpanBuilder(Parcel parcel) {
        span = new AbsoluteSizeSpan(parcel);
    }

}

File: span-builder/src/main/java/com/maciega/bartosz/spanbuilder/spans/builders/AbsoluteSizeSpanBuilder.kt
package com.maciega.bartosz.spanbuilder.spans.builders

import android.os.Parcel
import android.text.style.AbsoluteSizeSpan

/**
 * Created by bartoszmaciega on 03/03/17.
 */

class AbsoluteSizeSpanBuilder : AbstractSpanTypeBuilder<AbsoluteSizeSpan> {

  constructor(size: Int, dip: Boolean) {
    span = AbsoluteSizeSpan(size, dip)
  }

  constructor(parcel: Parcel) {
    span = AbsoluteSizeSpan(parcel)
  }

}
